src/main/java/me/youhavetrouble/purpurextras/config/PurpurConfig.java

```diff
@@ -23,9 +25,10 @@
             dispenserBreakBlockPickaxe, dispenserBreakBlockShovel, dispenserBreakBlockHoe, dispenserBreakBlockShears,
             dispenserBreakBlockAxe, grindstoneGivesEnchantsBack, dispenserShearPumpkin, dispenserActivatesJukebox,
             upgradeWoodToStoneTools, upgradeStoneToIronTools, upgradeIronToDiamondTools, requireNametagForRiding,
-            noTargetPermissions;
+            noTargetPermissions, stonecutterBlacklistEnabled;
     public final String beeHiveLoreBees, beeHiveLoreHoney;
     public final HashMap<Material, Material> anvilCrushBlocksIndex = new HashMap<>();
+    public final HashMap<EntityType, Boolean> stonecutterBlacklist = new HashMap<>();
```

Comment:
I'm not sure if I did this correctly because `Boolean` part won't be used anyway.

Reply:
did you try HashSet? ;)

Reply:
Thanks. I changed HashMap to HashSet, and it is beautiful.